script_editor/app/views/plays/show.html.erb

```diff
@@ -125,6 +125,88 @@
             <!-- This button shows the play's analytics -->
             <button class="nav-analytics" id="analytics-button">ANALYTICS</button>
 
+            <!-- This button toggles the strikethrough edits' visibility -->
+             <!-- Not sure which one to use for display in the else statement, initial or inherit -->
+            <button class = "nav-synopsis" curState = "showing" id="viewToggle-button"> Hide Edits </button>
+
+            <script>
+                var mainScript = document.getElementsByClassName("script-main");
+                var toggleBtn = document.getElementById("viewToggle-button");
+                toggleBtn.curState = "showing";
+                toggleBtn.onclick = function() {
+                    console.log(mainScript);
+                    var words = document.getElementsByClassName("word");
```

Comment:
you can make this section all one array, meaning we don't need three for loops, by using `array1.concat(array2).concat(array3)` The one line may be uglier, but we wont have to repeat each for loop